_studio/shared/umc/codec/h264_dec/src/umc_h264_dec_defs_yuv.cpp

```diff
@@ -86,21 +90,26 @@ void H264DecYUVBufferPadded::Init(const VideoDataInfo *info)
     m_pVPlane = 0;
     m_pUVPlane = 0;
 
-    if (m_chroma_format > 0)
+    if ((m_chroma_format > 0) && (info->GetNumPlanes() >= 2))
     {
         m_chromaSize = info->GetPlaneInfo(1)->m_ippSize;
     }
     else
     {
-        m_chromaSize.width = 0;
-        m_chromaSize.height = 0;
+        m_chromaSize = {0, 0};
     }
 }
 
 void H264DecYUVBufferPadded::allocate(const FrameData * frameData, const VideoDataInfo *info)
 {
     VM_ASSERT(info);
     VM_ASSERT(frameData);
+    VM_ASSERT(info->GetNumPlanes());
+    if (info == nullptr || frameData == nullptr || info->GetNumPlanes() == 0)
+    {
+        deallocate();
```

Comment:
that's a new code... Looks to be a separate bug fix?

Reply:
no, it's relate to fix static analysis issues.